coalib/parsing/FilterHelper.py

```diff
@@ -59,9 +60,9 @@ def apply_filters(filters, bears=None, sections=None):
     and runs filter in bear filtering mode.
 
     :param filters:
-        List of args based on ``bears`` has to be filtered. For example:
-        ``[['language', 'c', 'java'], ['can_fix', 'syntax'],
-        ['section_tags', 'save']]``
+        Dict of args based on ``bears`` has to be filtered. For example:
+        ``{'language': ('c', 'java'), 'can_fix': 'syntax',
```

Comment:
Docs need update, please change to `('syntax',)`; https://github.com/coala/coala/pull/5707/commits/721aabda8accf955ba01be090339b405f0dfbc10#diff-0ba65c8ebc811938f8e34a547109a908R105